src/ast/mod.rs

```diff
@@ -2029,6 +2029,7 @@ pub enum Statement {
         if_not_exists: bool,
         /// if true, has SQLite `TEMP` or `TEMPORARY` clause <https://www.sqlite.org/lang_createview.html>
         temporary: bool,
+        to: Option<ObjectName>,
```

Comment:
Could we add the description and a link to the docs where the operation comes from?

Reply:
Okay, I added a brief description and a link to Clickhouse views